src/Dfc.CourseDirectory.WebV2/ServiceCollectionExtensions.cs

```diff
@@ -161,6 +160,7 @@ public static class ServiceCollectionExtensions
             services.Configure<GoogleTagManagerOptions>(configuration.GetSection("GoogleTagManager"));
             services.AddTransient<SqlDataSync>();
             services.AddScoped<RouteValuesHelper>();
+            services.AddTransient<Features.AddTLevel.Details.CommandValidator>();
```

Comment:
Why are we making this injectable? Seems like a deviation from the established pattern.

Reply:
It's a side-effect of how the Date validation stuff works (it needs access to ActionContext currently). I believe there's a comment to fix this up when the tag helper library supports adding validation some other way (as soon as I figure that out ;-)).